brat/test/golden/error/toplevel-leftovers2.brat.golden

```diff
@@ -0,0 +1,9 @@
+Error in test/golden/error/toplevel-leftovers2.brat@FC {start = Pos {line = 2, col = 8}, end = Pos {line = 2, col = 9}}:
+f(x) = x
+       ^
+
+  Type mismatch when checking (x,「x」)
```

Comment:
Where is this `(x,「x」)` coming from?

Reply:
Not sure, but that doesn't look right! My guess is that `(x, 「x」)` is the LHS and RHS of this clause